### ISSUE TITLE
Clear/hide specific notification

### ISSUE THREAD

**Issue by karelz** [tootsuite/mastodon#1900]
Clearing all notifications is awesome, sometimes you want to keep a few / clear (or hide) those you already reacted to (e.g. you already replied).

Maybe special icon per notification?
Another option is checkbox next to each notification, but it feels heavy-weight.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by magicalskye** [comment]
this is important for me too, when i get replies that cause me bad anxiety i can't really look at it without feeling bad, but it doesn't justify silencing the whole person, so i avoid looking at mastodon altogether because that thing is still there

**Comment by Cassolotl** [comment]
I've seen folks post things like "I've got an awful reply in my mentions - can you reply to me with cat pictures or whatever to bump it down the list?" So I think there's a clear need for this feature.

**Comment by Cassolotl** [comment]
For those who specifically find unpleasant mentions a problem, this issue might be worth thumbs-upping and subscribing to and commenting on:

Hide a specific status #1669

**Comment by trwnh** [comment]
This is available in the API for a while, just not in the web app

**Comment by mrgan** [comment]
I agree that the ability to hide/"archive" a reply—or a single status, as in #1669—would be great.

I used to use a Twitter client for a company account, to handle support requests etc. Since this was a multi-user app, and the point was to "handle" tweets and then file them away as closed, the app had an "archive" feature. My daily flow was to read through what came in, answer those that needed an answer, and archive the rest—much like email.

Occasionally, abusive or needlessly argumentative posts would come in. With those that truly crossed the line, muting or blocking the account was an option. But with those that were merely angry, redundant, or e.g. unfunny, archiving took it off my mind. There's an empowering feeling in getting yet another boneheaded reply and simply hiding it, without necessarily ending all communication with that person forever (which block/mute would do).

**Comment by jose-dvm** [comment]
I would love to see this implemented, now I don't have the better experience when I'm involved in different conversations.

Does someone knows if this is already done for mobile app?

I think a swipe gesture triggering a confirmation dialog would be OK, I'm not sure about accesibility though.
